usaspending_api/download/v2/download_count.py

```diff
@@ -53,14 +67,32 @@ def post(self, request):
             filter_query = QueryWithFilters.generate_transactions_elasticsearch_query(filters, **options)
             search = TransactionSearch().filter(filter_query)
             total_count = search.handle_count()
+        else:
+            options = {}
+            time_period_obj = AwardSearchTimePeriod(
+                default_end_date=settings.API_MAX_DATE, default_start_date=settings.API_SEARCH_MIN_DATE
+            )
+            new_awards_only_decorator = NewAwardsOnlyTimePeriod(
+                time_period_obj=time_period_obj, query_type=_QueryType.AWARDS
+            )
+            options["time_period_obj"] = new_awards_only_decorator
+            filter_query = QueryWithFilters.generate_awards_elasticsearch_query(filters, **options)
+            search = AwardSearch().filter(filter_query)
+            total_count = search.handle_count()
 
         if total_count is None:
             total_count = 0
 
         result = {
+            # TO-DO: the following fields that follow the transaction_ naming convention will be removed
```

Comment:
```suggestion
            # TODO: the following fields that follow the transaction_ naming convention will be removed
```

Minor recommended change because most IDEs will auto highlight `TODO`